Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Armoury.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class Armoury : MonoBehaviour
+{
+    public Player player;
+    public PointsSystem currency;
+
+    IList<int> GetBuyableUnits; //Change to UnitType once Units are in
```

Comment:
These should be functions which take the player as a param and returning a List<int>. There should be checks in place to see if the units are buyable at this point in time (see the design doc). There's a seperate object which is a list of buyableUnits/buyableCards as per the UML diagram